api/src/routers/meals.js

```diff
@@ -0,0 +1,114 @@
+import express from 'express';
+import knex from '../database_client.js'; 
+
+const mealsRouter = express.Router();
+
+
+  mealsRouter.get("/", async (req, res) => {
+    try {
+      const meals = await knex("Meal").select("*"); // Select all columns from the Meal table
+      res.json(meals);
+    } catch (error) {
+      console.error("Error retrieving meals:", error);
+      res.status(500).json({ error: "Error retrieving meals from the database." });
```

Comment:
It would be nice to try to move this logic to a middleware

Reply:
I did it